mio/models/process.py

```diff
@@ -61,7 +61,7 @@ class NoisePatchConfig(BaseModel):
         default=True,
         description="Whether to use patch based noise handling.",
     )
-    method: str = Field(
+    method: Literal["mean_error", "gradient"] = Field(
```

Comment:
this how you specify that a value can only have specific values btw, so this way we don't need to re-do this validation everywhere we might use it, we know for sure it will always be one of these.